osf/migrations/0209_auto_20200701_1658.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-07-01 16:58
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0208_update_EGAP_schema'),
```

Comment:
This migration shouldn't be here. It should be after 0211_auto_20200709_1320.py.

Reply:
Sorry, I had master checked out. It should be after 0213_auto_20200728_1609.py.